pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -142,13 +144,22 @@ router.get('/*', (req, res, next) => {
                 callback(null);
             } else {
                 debug('Read job sequence');
-                serverJobs.getJobSequence(fileEdit.jobSequenceId, res.locals.course.id, (err, job_sequence) => {
+                serverJobs.getJobSequenceWithFormattedOutput(fileEdit.jobSequenceId, res.locals.course.id, (err, job_sequence) => {
                     if (ERR(err, callback)) return;
                     fileEdit.jobSequence = job_sequence;
                     callback(null);
                 });
             }
         },
+        (callback) => {
+            const promisified = callbackify(editorUtil.getErrorsAndWarningsForFilePath);
```

Comment:
I'm a bit confused about the name here. I would have thought we'd call this `callbackified`? And use `promisfied` in the case that we did `const promisified = util.promisify(...)`?

Reply:
Yep, this is just 100% wrong - just fixed this.